### DISCUSSION TITLE
[Feature] Facial recognition bulk corrections and date of birth dependancy

### DISCUSSION THREAD

**Discussion by Vlayke** [immich-app/immich#12823] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'v got two children. Both girls. One was born a month ago and another is three years old.
It's not surprising that Immich will misstag some.
But what I'd like is two things:
- Ability to select all the photos where younger daughter is incorrectly tagged and replace the tag to the older daughter.
- Ability for Immich to understand that if photo is older than date of birth of the person, then it should re-scan and re-identify those photos to see if it made a mistake.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#12460/#4743
